### PR TITLE
Add automated tests with Molecule

### PR DESCRIPTION
**Pull request by ajdecon**
Adds some basic automated tests with [Ansible Molecule](https://molecule.readthedocs.io/en/latest/index.html), as well as a Github action to execute them on push and PR.

Also makes some minor changes to the role to get the tests passing. 😉

### CODE REVIEW

**Review by lukeyeager — APPROVED** [review]
Looks slick! https://github.com/ajdecon/ansible-role-enroot/actions/runs/1655877404

File: molecule/default/molecule.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+platforms:
+  - name: ubuntu-1804
+    image: geerlingguy/docker-ubuntu1804-ansible
+    pre_build_image: true
+  - name: ubuntu-2004
+    image: geerlingguy/docker-ubuntu2004-ansible
+    pre_build_image: true
+  - name: centos-7
+    image: geerlingguy/docker-centos7-ansible
+    pre_build_image: true
```
Comment (lukeyeager) [inline molecule/default/molecule.yml:15]: Could do centos8 too, right? It's listed in `meta/main.yml`, at least.

**Comment by ajdecon** [comment]
Added CentOS 8! And confirmed that all tests pass on this branch. 🎉 https://github.com/ajdecon/ansible-role-enroot/runs/4717679877

File: tasks/redhat.yml [hunk 2]
```diff
@@ -41,29 +46,36 @@
       state: "absent"
     when:
     - "'enroot' in ansible_facts.packages"
-    - "ansible_facts.packages['enroot'][0]['version'] != enroot_version_string"
+    - "ansible_facts.packages['enroot'][0]['version'] != enroot_version_string.split('-')[0]"
   - name: enroot rpm packages
     yum:
       name: "{{ item }}"
       state: "{{ enroot_package_state }}"
+      disable_gpg_check: true
```
Comment (kftsehk) [inline tasks/redhat.yml:54]: fixes #19 #17

**Pull request merged**